### PR TITLE
Add blog post calling people for instructor training

### PR DESCRIPTION
**Pull request by rgaiacs**
<del>**Please wait https://github.com/swcarpentry/board/pull/53**.</del>

Ready to merge.

### CODE REVIEW

**Comment by gvwilson** [comment]
Why the changes to `Makefile` ?

**Comment by rgaiacs** [comment]
> Why the changes to Makefile ?

Trying something at my machine. I reverted the change on `Makefile`, squashed and rebased this pull request.

File: blog/2015/10/call-for-instructor-training.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+root: ../../..
+author: Raniere Silva
+title: "Call for Applications to December Instructor Training"
+date: 2015-10-20
+time: "22:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+As we <a href="{{page.root}}/blog/2015/09/rebooting-instructor-training.html">announced previous</a>,
```
Comment (gvwilson) [inline blog/2015/10/call-for-instructor-training.html]: s/previous/previously/

File: blog/2015/10/call-for-instructor-training.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+we will run another round of the free/open Instructor Training in December.
+Is time for people interested in participate at this round apply for the
```
Comment (gvwilson) [inline blog/2015/10/call-for-instructor-training.html]: People who are interested in participating in this round can now apply to take part.

File: blog/2015/10/call-for-instructor-training.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+<p>Students <strong>must</strong> apply in groups that are able to co-locate in one room for the duration of the Instructor Training. And each group should commit to jointly run one workshop no later than April 1st, 2016. <strong>Please note: how to run a workshop is covered in the curriculum.</strong></p>
+<p>Each student is expected to:</p>
+<ul>
+  <li><p>Be on the current waiting list or demonstrate sufficient competence with core tools used in the lessons they will teach.</p></li>
+<li><p>Watch one of the two videos below and submit a paragraph of feedback on the video they chose to watch as part of their application.</p>
+<ul>
+<li>https://vimeo.com/139316669</li>
+<li>https://vimeo.com/139181120</li>
+</ul></li>
+<li><p><strong>Only</strong> students interested in teaching for Software Carpentry need to submit a pull request to the specified GitHub repository.</p>
```
Comment (gvwilson) [inline blog/2015/10/call-for-instructor-training.html]: I would change this to, "People who wish to teach Software Carpentry must submit a small pull request to a specified GitHub repository.  People who wish to teach Data Carpentry do not need to do so."  (And have you spoken to DC to make sure they've reviewed this doc and these rules?)

File: blog/2015/10/call-for-instructor-training.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<li><p>The group's plan for the follow-up workshop.</p></li>
+</ul>
+<p>Groups will <strong>not</strong> be selected based on:</p>
+<ul>
+<li><p><strong>Affiliation with a particular institution, lab, or university.</strong> All groups will be considered equally.</p></li>
+<li><p><strong>Academic qualification.</strong> Faculty and students will be considered equally, as will groups from different academic disciplines.</p></li>
+</ul>
+<h3 id="timeline">Timeline</h3>
+<ul>
+<li>2015-10-20: blog post announcing the selection goes online.</li>
```
Comment (gvwilson) [inline blog/2015/10/call-for-instructor-training.html]: Yeah, well, that didn't happen :-)

**Comment by rgaiacs** [comment]
@jduckles Please only merge this after talk with @tracykteal to check if Data Carpentry want to publish something similar.

**Comment by jduckles** [comment]
@tracykteal let me know how you'd like to proceed, we should probably get this posted at the latest Monday given the timeline.

**Comment by jduckles** [comment]
@rgaiacs Can you change the blog post date to today and I'll merge later tonight, but give @tracykteal a bit to respond.

File: blog/2015/10/call-for-instructor-training.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+<li><p>The group's plan for the follow-up workshop.</p></li>
+</ul>
+<p>Groups will <strong>not</strong> be selected based on:</p>
+<ul>
+<li><p><strong>Affiliation with a particular institution, lab, or university.</strong> All groups will be considered equally.</p></li>
+<li><p><strong>Academic qualification.</strong> Faculty and students will be considered equally, as will groups from different academic disciplines.</p></li>
+</ul>
+<h3 id="timeline">Timeline</h3>
+<ul>
+<li>2015-10-22: blog post announcing the selection goes online.</li>
```
Comment (jduckles) [inline blog/2015/10/call-for-instructor-training.html]: Change this line as well.

**Comment by rgaiacs** [comment]
> Can you change the blog post date to today

Done.

> and I'll merge later tonight.

Thanks.

**Comment by rgaiacs** [comment]
@jduckles I added the hyperlinks and pushed upstream.

**Pull request merged**